setup.py

```diff
@@ -23,5 +26,5 @@
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.3',
-    ]
-)
+        ]
```

Comment:
These indent changes dont appear to add any benefit for style or compliance with guidelines...?